.github/workflows/build_example.yml

```diff
@@ -27,6 +27,15 @@ jobs:
           - idf_ver: "release-v4.4"
             parallel_count: 1
             parallel_index: 1
+          - idf_ver: "release-v5.1"
+            parallel_count: 1
+            parallel_index: 1
```

Comment:
With parallel_count=1 this job will probably be very slow. We can merge it as it is now, though